Cargo.toml

```diff
@@ -134,7 +134,7 @@ uuid = { version = "1", default-features = false }
 wascap = { version = "0.11", default-features = false }
 wasi-common = { version = "13", default-features = false }
 wasmcloud-actor = { version = "*", path = "./crates/actor" }
-wasmcloud-actor-derive = { version = "*", path = "./crates/actor/derive" }
+wasmcloud-actor-macros = { version = "*", path = "./crates/actor/derive" }
```

Comment:
Could you also move the directory to `crates/actor/macros` ?

Reply:
@rvolosatovs check this out